geomesa-tutorials-common/src/main/java/org/geomesa/example/transformations/GeoMesaQueryTutorial.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+package org.geomesa.example.transformations;
```

Comment:
Any idea why the diff is showing this as a complete diff and re-creation instead of a file name change?

Reply:
i think that if enough things change, it doesn't matter whether you do a `git mv` or not, it will treat it as a new file